src/main/kotlin/org/web3j/deploy/DeployPlugin.kt

```diff
@@ -0,0 +1,22 @@
+package org.web3j.deploy
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class DeployPlugin: Plugin<Project> {
+    override fun apply(project: Project) {
+
+        val extension = project.extensions.create("deploy", DeployExtension::class.java)
+
+        val task = project.tasks.create("deploy") {
+            it.doLast {
+                println("${extension.profileName} from ${extension.networkName}")
+                println("Hello from Deploy Plugin")
+                findDeployer("network-1", "org.web3j")
```

Comment:
We obviously can't hard code this, but assume it's testing code.. how can we detect correct package and network from running a gradle task, any thoughts?